client/src/components/HotTopics.jsx

```diff
@@ -175,7 +191,9 @@ export default class HotTopics extends React.Component {
         radarAnzeigen: "Microsoft",
         ring: msJSON[k].ring,
         ringAnzeigen: msJSON[k].ring.charAt(0).toUpperCase() + msJSON[k].ring.slice(1),
-        teilnehmer: this.getTeilnehmer(this.state.commentsAllSorted, msJSON[k].name, "microsoft").length
+        teilnehmer: this.getTeilnehmer(this.state.commentsAllSorted, msJSON[k].name, "microsoft").length,
+          desc: msJSON[k].desc,
+          name: msJSON[k].name,
```

Comment:
warum hast du `name` hinzugefügt? das ist doch eigentlich genau das gleiche wie `technologie` ein paar zeilen drüber

Reply:
@cosmoem ah schwierig zu erklären.ja das habe ich schon verstanden ,dass  eigentlich technologie=name aber um  BlipSheetDetailComponent zu zeigen müssen die props genau so heissen wie in BlipSheetDetailComponent... sonst kriege ich ein Fehler dass einige props nicht definiert sind.  kann dir später zeigen was ich meine

Reply:
ah okay, verstehe.. dann wäre es wahrscheinlich sinnvoll einfach ´technologie´ durch `name` überall zu erstetzen damit wir nicht quasi die gleiche variable 2x haben, ist aber nicht so wichtig, aber beim code clean up könnten wir das machen